migtests/tests/pg/basic-public-live-test/validate

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import yb
+import collections
+
+def main():
+	parser = argparse.ArgumentParser(description='Process an argument.')
+	parser.add_argument('--live_migration', type=str, help='An argument for checking the if live migration is enabled or not')
+	parser.add_argument('--ff_enabled', type=str, help='An argument for checking the if fall-forward is enabled or not')
+	parser.add_argument('--fb_enabled', type=str, help='An argument for checking the if fall-back is enabled or not')
+	args = parser.parse_args()
+	global live_migration 
+	live_migration = args.live_migration
+	global ff_enabled
+	ff_enabled = args.ff_enabled
+	global fb_enabled
+	fb_enabled = args.fb_enabled
+
+	migration_completed_checks_yb()
+
+	if ff_enabled == 'true':
+		migration_completed_checks_pg()
+	
+	if fb_enabled == 'true':
+		migration_completed_checks_src()
+
+EXPECTED_ROW_COUNT = {
+	'x':3,
+ 	'user_table': 8
+}
+
+EXPECTED_DATA_TYPES = {
+	'x': {
+		'id': 'integer',
+		'id2': 'integer'
+	},
+	'user_table': {
+		'id': 'integer',
+		'email': 'character varying',
+		'status': 'character varying'
+	}
+}
+
+EXPECTED_SUM_OF_COLUMN = {
+	'x': {
+		'id': '6',
+		'id2': '9'
+	},
+	'user_table': {
+		'id': '36',
+	}
+}
+
+def migration_completed_checks_yb():
+	print("Running tests on YB")
+	yb.run_checks(migration_completed_checks)
+
+def migration_completed_checks_pg():
```

Comment:
rename these to `migration_completed_checks_ff` / `migration_completed_checks_fb` to be consistent with oracle ones.